Containerfile

```diff
@@ -36,24 +36,20 @@ WORKDIR "${APP_HOME}"
 # https://github.com/containers/podman-compose/issues/166
 # USER "${APP_USER}"
 
-COPY ./.ruby-version "${APP_HOME}"
-COPY ./Gemfile ./Gemfile.lock "${APP_HOME}"
+COPY ./.ruby-version "${APP_HOME}"/
+COPY ./Gemfile ./Gemfile.lock "${APP_HOME}"/
 COPY ./vendor/cache "${APP_HOME}"/vendor/cache
 
 # Fixes https://github.com/sass/sassc-ruby/issues/146
 RUN bundle config build.sassc --disable-march-tune-native
 
 RUN bundle check || bundle install --jobs 20 --retry 5
 
-COPY ./package.json ./yarn.lock ./.yarnrc "${APP_HOME}"
```

Comment:
I just quickly verified that `rails assets:precompile` will in fact run a `yarn install`, so this seems ok.